core/src/main/scala/org/apache/spark/status/api/v1/ApplicationListResource.scala

```diff
@@ -69,6 +69,8 @@ private[spark] object ApplicationsListResource {
       attempts = app.attempts.map { internalAttemptInfo =>
         new ApplicationAttemptInfo(
           attemptId = internalAttemptInfo.attemptId,
+          startTimeEpoch = internalAttemptInfo.startTime,
```

Comment:
Given there's no example of that in the api I (apparently wrongly) assumed that I couldn't add methods like getters to the api classes. I'll add them, test and push when I have time later today.